gatsby/lobid/src/components/team.html.js

```diff
@@ -19,26 +20,65 @@ export class Team extends React.Component {
     super(props);
     this.props = props;
     this.state = {
-      infoToggled: false
+      infoToggledMembers: false,
+      infoToggledProjects: false
     };
   }
 
-  toggleEhemalige = () => {
-    this.setState(prevState => ({ infoToggled: !prevState.infoToggled }));
+  toggleFormerMembers = () => {
+    this.setState(prevState => ({
+      infoToggledMembers: !prevState.infoToggledMembers,
+      infoToggledProjects: prevState.infoToggledProjects
+    }));
```

Comment:
just a small hint: you can use the spread operator here and only make the change to what you actually want to change. especially when states get bigger, you might not want to type them all out or one might forget one.
E.g.
```
{
  infoToggledMembers: !prevState.infoToggledMembers,
  ...prevState
}
```

see also: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
Ah, thanks, I didn't know about the spread syntax! Applied in https://github.com/hbz/lobid/pull/482/commits/f71eecc803fcac19e35a6a53b56ebc51f1eaedc9 (it seems the `...prevState` has to go first, which makes sense if I understand it correctly, since we basically create a copy of `prevState` and then replace one value?)

Reply:
of course, you're right!